lib/LocalDeepnet.js

```diff
@@ -511,13 +511,18 @@ LocalDeepnet.prototype.deepnetPredict = function (inputData) {
 
   var uniqueTerms, probabilities = [], total = 0, len, probability,
     category, yOut, order, index, probabilityInfo, inputArray, prediction,
-    distribution, sortedPrediction;
+    distribution, sortedPrediction, prediction = null;
   uniqueTerms = this.getUniqueTerms(inputData);
   inputArray = this.fillArray(inputData, uniqueTerms);
   if (typeof this.networks !== 'undefined') {
-    return this.predictList(inputArray);
+    prediction = this.predictList(inputArray);
+  } else {
+    prediction = this.predictSingle(inputArray);
+  }
+  if (typeof prediction != 'object') {
+    prediction = { prediction : prediction };
```

Comment:
If we change the prediction to return always an object, then we can remove the distinction and recover the old existing code, right?

Reply:
I mean, I think that the .toPrediction method is responsible to format the output of the predict method, so maybe we could build the object there.